src/dombeck_lab_to_nwb/azcorra2023/azcorra2023_metadata.yaml

```diff
@@ -13,18 +13,30 @@ NWBFile:
     neuron subtypes and demonstrate that molecular expression patterns can serve as a common framework to dissect
     dopaminergic functions.
   session_description: |
-    This session contains the acquisition traces (velocity from rotary encoder, trigger signals for reward, air puff
-    and light stimuli delivery, licking from a lick sensor, fluorescence and waveform generator output (used to
-    alternate 405-nm and 470-nm illumination every 10 ms) collected at 4000 Hz by Picoscope 6.
+    This session contains fiber photometry recordings from head-fixed mice running on a treadmill while receiving
+    rewards and aversive air puffs in order to analyze differences or similarities in functional responses of different
+    subtypes of striatonigral dopamine neurons.
+  keywords:
+    - Fiber photometry
+    - GCaMP6f
+    - Dopamine
   institution: Northwestern University
   lab: Dombeck
   experimenter:
     - Azcorra, Maite
-
-Ecephys:
-  ElectricalSeriesVelocity:
-    description: The electrical series of the velocity from rotary encoder collected at 4000 Hz by Picoscope.
-  ElectricalSeriesFluorescenceRed:
-    description: The electrical series of fluorescence from Red channel collected at 4000 Hz by Picoscope.
-  ElectricalSeriesFluorescenceGreen:
-    description: The electrical series of fluorescence from Green channel collected at 4000 Hz by Picoscope.
+  virus: |
```

Comment:
I would put this description under pharmacology and use the field virus for reporting just the virus notation (AAV8-EF1α-CreOn/FlpOn-GCaMP6f (RRID:Addgene_137122, titer 6.10 × 1013)
    for Aldh1a1-iCre/Th-Flpo, Vglut2-IRES-Cre/Th-Flpo and Calb1-IRES2-Cre/Th-Flpo mice or AAV1-CAG-FLEX-GCaMP6f (RRID:Addgene_100835, titer 2.00 × 1013) for DAT-Cre or Anxa1-iCre mice)